rescript/subsample.py

```diff
@@ -0,0 +1,47 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import random
+from itertools import tee
+from typing import Union
+from q2_types.feature_data import DNAIterator
+
+
+def subsample_fasta(sequences: DNAIterator,
+                    subsample_size: Union[float, int] = 0.1,
+                    random_seed: int = 42) -> DNAIterator:
```

Comment:
just set random seed default to 1... otherwise you will get questions from people who think there's some magic going on here!